src/sentry/static/sentry/app/views/organizationHealth/browsers.jsx

```diff
@@ -0,0 +1,98 @@
+import {Flex} from 'grid-emotion';
+import React from 'react';
+
+import {t} from 'app/locale';
+import PieChart from 'app/components/charts/pieChart';
+
+import EventsTableChart from './eventsTableChart';
+import Header from './styles/header';
+import HealthPanelChart from './styles/healthPanelChart';
+import HealthRequest from './util/healthRequest';
+import withHealth from './util/withHealth';
+
+class OrganizationHealthBrowsers extends React.Component {
+  render() {
+    let {className} = this.props;
```

Comment:
Do we need to pass classname here? I think it'd be good to avoid encouraging usage of classnames since we're doing styled components and excessively passing props around if not required

Reply:
Oops forgot to remove this.

Just to clarify though, styled components and classNames usage is not mutually exclusive. We would need `className` if we wanted `styled` to be able to extend `OrganizationHealthBrowsers`.